src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/interior_data_cache_impl.cc

```diff
@@ -69,4 +82,27 @@ void InteriorDataCacheImpl::ClearCache() {
   sync_primitives::AutoLock autolock(subscriptions_lock_);
   subscriptions_.clear();
 }
+
+void InteriorDataCacheImpl::ResetRequestCountOnTimer() {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  bool is_alive = true;
+  while (is_alive) {
+    std::this_thread::sleep_for(
```

Comment:
@ZhdanovP timer is not a thread . I mean that do not implement cycle in timer. Timer implements cycle for you. Just reset amount of requests in timer